tests/IterableCodeExtractorTest.php

```diff
@@ -2,16 +2,16 @@
 
 namespace WP_CLI\I18n\Tests;
 
-use PHPUnit_Framework_TestCase;
 use WP_CLI\I18n\IterableCodeExtractor;
 use WP_CLI\Utils;
+use Yoast\PHPUnitPolyfills\TestCases\TestCase;
 
-class IterableCodeExtractorTest extends PHPUnit_Framework_TestCase {
```

Comment:
@schlessera I'm getting `PHP Fatal error:  Uncaught Error: Class 'PHPUnit_Framework_TestCase' not found` on `master` without this change. Any advice?

Reply:
Use `WP_CLI\Tests\TestCase` instead, to avoid direct coupling to the polyfills.